.eslintrc.js

```diff
@@ -1 +1,6 @@
-module.exports = require('eslint-config-wolox-node');
+module.exports = {
+  extends: 'eslint-config-wolox-node',
+  rules: {
+    'no-console': 'off'
```

Comment:
I'll leave it as before.
I changed it because had this warning with sequelize logger :
`(node:47619) [SEQUELIZE0002] DeprecationWarning: The logging-option should be either a function or false. Default: console.log`